jennchenn/uwexplore
# Added ability to add/delete courses from the calendar

## Implementation description

- Can add courses from search cards
- Can delete courses from the onClick modal
- Can change the course colours

Limitations:
- Cannot delete a course from the search cards yet 
- Cannot choose which section to add to the calendar
- After deleting a course, some colours may change

- Resolves BREAD-91, BREAD-64



## Steps to test

1. search for syde courses and add one to the calendar - verify at least one section appears on the schedule 
2. click on one of the courses on the calendar, then delete it - verify it dissapears
3. change the color on one of the classes - verify it changes, and that the color selector reflects the correct colors



## What should reviewers focus on?

- Which one of the limitations should be worked on first? Any need to be done before heuristic evaluation?
